tests/test_columns.py

```diff
@@ -73,6 +77,10 @@ async def test_model_crud():
     product = await Product.objects.get(pk=product.pk)
     assert product.created.year == datetime.datetime.now().year
     assert product.created_day == datetime.date.today()
+    assert product.created_date.today() == datetime.date.today()
+    assert product.created_datetime.year == datetime.datetime.now().year
```

Comment:
I think the date time checks aren't very accurate, it's only checking the year part.
The best way to deal with that is probably to mock the date time in the test so it's always fixed to a value.
Or at least we could check the date part, not year only.